### DISCUSSION TITLE
Slides for Quiz 4.3 request or hints

### DISCUSSION THREAD

**Discussion by bonosa** [Lightning-AI/dl-fundamentals#20] [Lecture videos and quizzes] [answered]
Hi Sebastian, I’m rewatching the lectures in unit 4.3 and missing the calculation for the number of parameters. Could you please post a slide or give me hints or specify which of the unit 4 videos the calculation is in? I googled but didn’t find precisely the correct answer. Thanks

**Comment by rasbt** [comment, marked as answer]
Hey there! 
Sorry, there is no explicit calculation in the slides -- I didn't want to make it too easy. But as a hint, all the necessary information to compute the number of parameters in a given layer is shown in this figure.

If you have a neural network with multiple layers, you compute the number of parameters for each layer and then add these numbers up. 

I am happy to give more help if you get stuck, but maybe this is already enough information to help you solve it :)
<img width="579" alt="Screenshot 2023-01-27 at 8 45 47 AM" src="https://user-images.githubusercontent.com/5618407/215114966-faa1e86c-4565-4528-b2f4-95c6a253d8a4.png">

**Comment by bonosa** [comment]
Yes!!! Thanks I get it now!😃
